packages/bpk-component-switch/src/BpkSwitch.tsx

```diff
@@ -27,44 +23,34 @@ import STYLES from './BpkSwitch.module.scss';
 const getClassName = cssModules(STYLES);
 
 export type Props = {
-  label: Node,
-  className: ?string,
+  ariaLabel: string;
+  className?: string | null;
+  small?: boolean;
+  [rest: string]: any;
 };
 
-const BpkSwitch = (props: Props) => {
-  const { className, label, small, ...rest } = props;
-
+const BpkSwitch = ({
+  ariaLabel,
+  className = null,
+  small = false,
+  ...rest
+}: Props) => {
   const switchClassNames = getClassName(
     'bpk-switch__switch',
     small && 'bpk-switch__switch--small',
   );
 
   return (
     <label className={getClassName('bpk-switch', className)}>
```

Comment:
does this still need to be wrapped in a label?

Reply:
Taking a look yes it still needs to be a label so that the component is still fully selectable and operational as a switch with a Assistive Tech, otherwise looks like this

![Screenshot 2024-08-12 at 13 16 45](https://github.com/user-attachments/assets/7ee2bc91-b37d-4191-8c9f-869b52df6d1b)